src/services/load_from_xml_manager.py

```diff
@@ -728,12 +728,13 @@ def load_door_from_save(door, gap_x, gap_y):
     return loaded_door
 
 
-def load_building_from_save(building, gap_x, gap_y):
+def load_building_from_save(building, gap_x, gap_y, shop_balance=None):
```

Comment:
Let's put some default numeric value instead of `None` for now, maybe `500` as in other places?

I would like to avoid `None`, because then it means somewhere else in the application, we would have to handle this None case to avoid crashes (because `shop.shop_balance -= item.price` would lead to unclear behaviour if shop_balance is None for example).